apps/mobile/src/screens/HomeScreen/TwitterSuggestionListScreen.tsx

```diff
@@ -0,0 +1,97 @@
+import { useNavigation } from '@react-navigation/native';
+import { Suspense, useCallback, useMemo } from 'react';
+import { SafeAreaView, ScrollView, TouchableOpacity, View } from 'react-native';
+import { graphql, useLazyLoadQuery } from 'react-relay';
+
+import { XMarkIcon } from '~/components/Search/XMarkIcon';
+import { LoadingFollowerList } from '~/components/Trending/LoadingFollowerList';
+import { SuggestionUser } from '~/components/Trending/SuggestionUser';
+import { Typography } from '~/components/Typography';
+import { TwitterSuggestionListScreenQuery } from '~/generated/TwitterSuggestionListScreenQuery.graphql';
+
+export function TwitterSuggestionListScreen() {
+  const query = useLazyLoadQuery<TwitterSuggestionListScreenQuery>(
+    graphql`
+      query TwitterSuggestionListScreenQuery($twitterListFirst: Int!, $twitterListAfter: String) {
+        socialConnections(
+          after: $twitterListAfter
+          first: $twitterListFirst
+          socialAccountType: Twitter
+          excludeAlreadyFollowing: false
+        ) {
+          edges {
+            node {
+              __typename
+              ... on SocialConnection {
+                __typename
+                galleryUser {
+                  ... on GalleryUser {
+                    __typename
+                    ...SuggestionUserFragment
+                  }
+                }
+              }
+            }
+          }
+          pageInfo {
+            total
+          }
+        }
+        ...SuggestionUserQueryFragment
+      }
+    `,
+    {
+      twitterListFirst: 24,
+      twitterListAfter: null,
+    }
+  );
```

Comment:
@tbezman any idea how do I get pagination from `useLazyLoadQuery`? 

For context, we want to make this modal infinite scroll just like in the web